tests/bindings/nodejs/HawkTracerClient.test.ts

```diff
@@ -6,10 +6,10 @@ const mockNativeClientClass = jest.fn();
 const mockNativeClient = jest.fn();
 
 // @ts-ignore
-mockBindings.mockReturnValue({"HawkTracerClient": mockNativeClientClass});
+mockBindings.mockReturnValue({ "HawkTracerClient": mockNativeClientClass });
```

Comment:
These formatting changes seem unnecessary.

Reply:
Done

Reply:
I still see the format changes, although I won't insist on this too much.